packages/website/src/docs/toggle-button.mdx

```diff
@@ -1,17 +1,30 @@
 ---
-title: Toggle Button
-component: ToggleButton
+title: Toggle Button Group
+component: ToggleButtonGroup
+description: Useful for allowing users to select only one option from a set.
```

Comment:
should this be "to select 0, 1 or more options from a set"?

Reply:
Oh yeah, forgot I added that feature :D